handlers/main.yml

```
@@ -0,0 +1,18 @@
+---
+
+- name: Reload systemd daemon
+  systemd:
+    daemon_reload: yes
+  listen: "Restart pretix serivces"
+
+
+- name: Restart pretix
+  service:
+    name: '{{ item }}'
+    state: restarted
+    enabled: yes
+  with_items:
```

`with_items` is deprecated, please use `loop` instead
```suggestion
  loop:
```